src/many-ledger/src/main.rs

```diff
@@ -90,6 +95,8 @@ fn main() {
     } else if persistent.exists() {
         // Initial state is ignored.
         state = None;
+    } else if snapshots.exists() {
```

Comment:
I think if snapshot root does not exist we should use the same as persistent's parent dir. WDYT?